### PR TITLE
first-login UI updates

### PR DESCRIPTION
**Pull request by KfirMor**
first-login UI cosmetic updates

### CODE REVIEW

File: src/app/pages/lesson/dialogs/new-edit/edit-lesson.dialog.scss [hunk 1]
```diff
@@ -64,19 +64,23 @@
...
 
-.mat-dialog-content {
+/deep/ .mat-dialog-content {
   line-height: 1.31;
   font-size: 16px;
   font-weight: normal;
   letter-spacing: 1.3px;
   height: 1005px;
+  overflow-y: hidden !important;
+  // padding-left: 0px;
```
Comment (eran1205) [inline src/app/pages/lesson/dialogs/new-edit/edit-lesson.dialog.scss]: Hi dude,

Please remove comment.

File: src/app/pages/shared/session-mgmt/session-footer/session-footer.component.html [hunk 2]
```diff
@@ -1,5 +1,7 @@
 <div class="session-mgmt-footer">
-  <img src='assets/img/msw-login-dec.png'/>
-  <div class="seperator"></div>
+  <div class ="msw-login-footer" >
+    <img class="img" src="/assets/img/tinti.png" alt="" >
```
Comment (eran1205) [inline src/app/pages/shared/session-mgmt/session-footer/session-footer.component.html]: Why we need the "alt" it it's empty?

File: src/app/pages/shared/session-mgmt/session-footer/session-footer.component.scss [hunk 3]
```diff
@@ -1,18 +1,33 @@
 :host {
   display: flex;
-  justify-content: center;
 
   .session-mgmt-footer {
     // padding: 0 10em;
```
Comment (eran1205) [inline src/app/pages/shared/session-mgmt/session-footer/session-footer.component.scss]: Please remove comment

Comment (eran1205) [reply src/app/pages/shared/session-mgmt/session-footer/session-footer.component.scss]: and all the class, it's empty

File: src/app/pages/lesson/dialogs/new-edit/edit-lesson.dialog.scss (hunk 1 above)
Comment (KfirMor) [reply src/app/pages/lesson/dialogs/new-edit/edit-lesson.dialog.scss]: not my code :-)
but I fixed it

File: src/app/pages/shared/session-mgmt/session-footer/session-footer.component.html (hunk 2 above)
Comment (KfirMor) [reply src/app/pages/shared/session-mgmt/session-footer/session-footer.component.html]: removed the "alt"

File: src/app/pages/shared/session-mgmt/session-footer/session-footer.component.scss (hunk 3 above)
Comment (KfirMor) [reply src/app/pages/shared/session-mgmt/session-footer/session-footer.component.scss]: not my code.. done

**Pull request merged**